lint.whitelist

```diff
@@ -574,8 +563,6 @@ CSS-COLLIDING-REF-NAME: css/css-masking/clip-path/reference/clip-path-square-002
 
 # Duplicate filename where there's no actual merging
 # (i.e., we're being more aggressive than the build system at forbidding it)
-CSS-COLLIDING-REF-NAME: css/CSS2/positioning/position-relative-001-ref.xht
```

Comment:
I see position-relative*.html deletions here, but not an xht one

Reply:
I didn't pay attention to the extension, but it appears as though the extension is ignored. I couldn't find where in the code the extension is stripped, but confirmed that the lint fails if these two lines are removed without making any other changes. Anyway, if the lint passes when a line of line.whitelist is removed, then that line can be removed :)